Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/Cards/TwitterCardPlayer.cs

```diff
@@ -92,15 +88,41 @@ protected override void Validate()
         {
             base.Validate();
 
+            if (string.IsNullOrEmpty(this.TwitterSiteUsername))
+            {
+                throw new ArgumentNullException(nameof(this.TwitterSiteUsername));
+            }
+
             if (this.Image == null)
             {
                 throw new ArgumentNullException(nameof(this.Image));
             }
 
+            if (string.IsNullOrEmpty(this.Title))
+            {
+                throw new ArgumentNullException(nameof(this.Title));
+            }
+
+            // ToDo: Add Check for Image.Alt if this.Player.StreamUrl is provided
             if (this.Player == null)
             {
                 throw new ArgumentNullException(nameof(this.Player));
             }
+
+            if (!this.Player.PlayerUrl.ToLower().Contains("https:"))
+            {
+                throw new ArgumentNullException(nameof(this.Player.PlayerUrl), "This must be a HTTPS URL which does not generate active mixed content warnings in a web browser.");
```

Comment:
I got an error with that syntax, but I'm sure I missed what you wanted me to do.  Currently, it's now; 
`throw new ArgumentNullException(nameof(this.Player.PlayerUrl), $"The " + nameof(this.Player.PlayerUrl) + " must be a HTTPS URL which does not generate active mixed content warnings in a web browser.");`